recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,444 @@
+import os
+import re
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class VTKConan(ConanFile):
+    name = "vtk"
+    # version = "9.0.1"  # DO NOT SUBMIT!!! Remove this line in final version of PR to conan-center-index
```

Comment:
Why do you need this line at all?

Just for the case that you don't know: You can provide the version number locally by executing e.g. `conan create . vtk/9.0.1@` in the recipe folder